frontends/web/src/routes/buy/pocket.tsx

```diff
@@ -57,27 +55,13 @@ export const Pocket = ({ code }: TProps) => {
   }, [config]);
 
   useEffect(() => {
-    onResize();
-    window.addEventListener('resize', onResize);
     window.addEventListener('message', onMessage);
 
     return () => {
-      window.removeEventListener('resize', onResize);
       window.removeEventListener('message', onMessage);
     };
   });
 
-  const onResize = () => {
-    if (resizeTimerID) {
-      clearTimeout(resizeTimerID);
-    }
-    resizeTimerID = setTimeout(() => {
-      if (!ref.current) {
-        return;
-      }
-      setHeight(ref.current.offsetHeight);
-    }, 200);
-  };
```

Comment:
how does this fix the double guide?

Reply:
I removed everything that has anything to do with setting the `height` in the `pocket` and `moonpay` file (I forgot to do so in Moonpay, will do it now). The reason for this is because with the new css changes, we don't use this dynamic value of `height` anymore to control the `div` height, but rather its height will always be `100%`.

Reply:
is this related to the double guide? I still don't see what fixes the 2x guide entries.

Reply:
What fixed it was actually the CSS changes. With the css changes, we don't need `height` to be set from the component anymore (height for both the div and iframe). Therefore, im removing it here.

 I first removed the setHeight, which is in the `onResize`. And it seems like `onResize` is only being used to `setHeight` therefore it has no use anymore once I removed `setHeight` - thus I removed it. Does it make sense? 🙂 

ofc we don't have to remove `onResize` but again, there seems to be no use of it anymore.